openapi-generator/go_lang.yaml

```diff
@@ -2,7 +2,7 @@
 generatorName: go
 outputDir: clients/go
 packageName: phrase
-packageVersion: 2.6.5
+packageVersion: 2.6.6
```

Comment:
is the version bump just due to tooling upgrade? should there be any actual change in the generated library?

Reply:
ah I see the comment now.